.github/workflows/sync-rest.yml

```diff
@@ -71,7 +71,7 @@ jobs:
             // TODO: The GraphQL API does have this info. Look into changing to that.
             const botForks = botRepos.flatMap(({ full_name, fork }) => fork ? [full_name.replace(process.env.BOT_USERNAME, 'exercism')] : [])
 
-            const tracks = ${{ steps.fetch-tracks.outputs.result }}
+            const tracks = ${{ steps.fetch-repos.outputs.result }}
```

Comment:
Should the JS variables also be renamed?

Reply:
I've renamed